gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -41,27 +35,11 @@ try:
 except AttributeError:  # pragma: NO COVER
     OptionalRetry = Union[retries.Retry, object, None]  # type: ignore
 
+from .rest_base import _Base{{ service.name }}RestTransport
```

Comment:
Two minor stylistic nits:
- can we move the `try: OptionalRetry...` block to after the imports?
- can we move the `from .restbase...` and `from .base...` lines to be next to each other and after the imports?

In other words:
```
import ...
import ...
from .rest_base ...
from .base ...

try:
  OptionalRetry = ...
```